server/project/manager.ts

```diff
@@ -218,34 +231,35 @@ export const createConcept = async (
 ) => {
   await validateProjectName(project);
 
-  concept = concept.toLowerCase();
-
-  if (!concept.endsWith(".ts")) {
-    concept += ".ts";
-  }
+  const conceptFileTitle = conceptFileName(concept);
```

Comment:
`filename` would work better instead of `conceptFileTitle`

Reply:
or the function can be called `convertToFilename(concept)` and this variable can be called `conceptFilename`.